pwndbg/wrappers/checksec.py

```diff
@@ -1,30 +1,16 @@
 from __future__ import annotations
 
-from subprocess import CalledProcessError
+from pwnlib.elf import ELF
 
-import pwndbg.commands
-import pwndbg.lib.cache
-import pwndbg.wrappers
 
-cmd_name = "checksec"
-cmd_pwntools = ["pwn", "checksec"]
+def get_raw_out(local_path: str) -> str:
+    elf = ELF(local_path)
```

Comment:
If you look over the `ELF` class with IPython via `ELF??` it keeps opened handles to some resources:
<img width="654" alt="image" src="https://github.com/pwndbg/pwndbg/assets/10009354/1e5ec1e9-acb1-42f7-aab0-a3e6b2c9a750">

The ELF object is also a context manager (implements `__enter__` and `__exit__`) so we should use it with a [`with` statement](https://docs.python.org/3/reference/compound_stmts.html#with) as follows:
```py
with ELF(path) as e:
    output += e.checksec()
```

Reply:
Thanks